pkg/kubectl/apply/doc.go

```diff
@@ -30,5 +30,5 @@ package apply
 //parsed, err := parseFactory.CreateElement(recorded, local, remote)
 //Expect(err).Should(Not(HaveOccurred()))
 //
-//merged, err := parsed.Accept(merge.Create(merge.Options{}))
+//merged, err := parsed.Merge(merge.Create(merge.Options{}))
```

Comment:
Really like it. It's unclear what is created by `merge.Create`? Should it be `merge.CreateStrategy`